src/client/bootstrap.tsx

```diff
@@ -124,9 +135,13 @@ const MetadataWrapper = () => {
 
   const props: ScalprumComponentProps<
     Record<string, any>,
-    { asyncData: { data: unknown } }
+    {
+      aadditionalData: { data: unknown };
```

Comment:
Is this supposedto be the `asyncData`?

Reply:
Good catch, thank you. Good ol mass rename got me